kevm_pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -37,6 +39,198 @@
 _LOGGER: Final = logging.getLogger(__name__)
 
 
+@dataclass
+class Contract():
+
+    @dataclass
+    class Method:
+        name: str
+        id: int
+        sort: KSort
+        arg_names: List[str]
+        arg_types: List[str]
```

Comment:
```suggestion
        arg_names: Tuple[str, ...]
        arg_types: Tuple[str, ...]
```

Reply:
Why do we prefer `Tuple[str, ...]` to `List[str]`?

Reply:
Is it because these are going to be set only once, so immutable representation is better? Also, does it type-check better?